pillar.example

```diff
@@ -1,5 +1,5 @@
 postgres:
-  pg_hba.conf: salt://postgres/pg_hba.conf
+  pg_hba.conf: salt://postgres/templates/pg_hba.conf.jinja
```

Comment:
Standard -formula practice is to put files in a files/ dir (vs templates/). See... every other formula that has files (collectd, rsyslog, graphite, etc). So far changing the actual extension of the file seems to be up to the author (there's no consensus, but more often than not it's not .jinja or whatever).